src/deluge/io/debug/print.cpp

```diff
@@ -110,7 +110,7 @@ void prependTimeStamp(bool isNewLine) {
 		lutHexString(Debug::readCycleCounter(), buffer);
 		buffer[8] = ' ';
 		buffer[9] = 0;
-		if (midiDebugDevice && SYSEX_LOGGING_ENABLED) {
+		if (midiDebugDevice && ENABLE_SYSEX_LOGGING) {
```

Comment:
this isn't needed, there's no issue always having this in the binary and we've taken advantage of it to troubleshoot issues with users before. Sysex logging enabling is handled by sending the logging enable sysex message

Reply:
Ok I will remove that flag